examples/BabelStream/standalone/babel_stream.py

```diff
@@ -12,23 +12,23 @@ def init_arrays(index: int, a: pk.View1D[pk.double], b: pk.View1D[pk.double], c:
     c[index] = initC
 
 @pk.workunit
-def copy(index: int, a: pk.View1D[pk.double], c: pk.View1D[pk.double]):
+def copy(index, a, c):
```

Comment:
Going Pythonic is nice, but now that we loose types, we should update the variable names to reflect something behind data being in those variables. If we do not want to use views any more, do we talk about arrays or lists?

Reply:
same comment for other functions/files

Reply:
In this case, I think these variable names are copied from the original C++/Kokkos examples, so we can keep it for now.

Reply:
Changed anyways